### PR TITLE
dev/reference/tools: update sync-diff-inspector.md

### PR DESCRIPTION
**Pull request by IzabelWang**
### What is changed, added or deleted? (required)
Update dev/reference/tools/sync-diff-inspector.md

### What is the related PR or file link(s)? (required) 
Via: [#1361](https://github.com/pingcap/docs-cn/pull/1361)

dev
v3.0
### Did you use Tab spaces in this PR? (required)
No

PTAL @lilin90

### CODE REVIEW

**Comment by morgo** [comment]
@IzabelWang Please list what versions this change effects and update v3.0 and v2.1 if relevant.

**Comment by IzabelWang** [comment]
Update sync-diff-inspector.md in v3.0.
PTAL @lilin90

File: v3.0/reference/tools/sync-diff-inspector.md [hunk 1]
```diff
@@ -34,7 +35,7 @@ log-level = "info"
 
 # sync-diff-inspector divides the data into multiple chunks based on the primary key,
 # unique key, or the index, and then compares the data of each chunk.
-# Use "chunk-size" to set the size of a chunk.
+Compare data in each chunk. Use "chunk-size" to set the size of a chunk.
```
Comment (lilin90) [inline v3.0/reference/tools/sync-diff-inspector.md]:
```suggestion
# Compare data in each chunk. Use "chunk-size" to set the size of a chunk.
```

File: dev/reference/tools/sync-diff-inspector.md [hunk 2]
```diff
@@ -46,10 +47,17 @@ sample-percent = 100
 # If disabled, data is compared line by line.
 use-checksum = true
 
-# If set to true, data check is ignored. If set false, data is checked.
-ignore-data-check = false
+# If it is set to true, data is checked only by calculating checksum. Data will not be checked after inspection, even if the upstream and downstream checksums are inconsistent.
```
Comment (lilin90) [inline dev/reference/tools/sync-diff-inspector.md]:
```suggestion
# If it is set to true, data is checked only by calculating checksum. Data is not checked after inspection, even if the upstream and downstream checksums are inconsistent.
```

File: v3.0/reference/tools/sync-diff-inspector.md [hunk 3]
```diff
@@ -47,10 +48,17 @@ sample-percent = 100
 # If disabled, data is compared line by line.
 use-checksum = true
 
-# If set to true, data check is ignored. If set false, data is checked.
-ignore-data-check = false
+# If it is set to true, data is checked only by calculating checksum. Data will not be checked after inspection, even if the upstream and downstream checksums are inconsistent.
```
Comment (lilin90) [inline v3.0/reference/tools/sync-diff-inspector.md]:
```suggestion
# If it is set to true, data is checked only by calculating checksum. Data is not checked after inspection, even if the upstream and downstream checksums are inconsistent.
```

**Pull request merged**